robot/rover/MobilePlatform/MobilePlatform.ino

```diff
@@ -481,3 +511,11 @@ void lb_encoder_interrupt(void) {
     LB.prevTime = micros();
     LB.encoderCount++;
 }
+
+void setIndicatorMode(uint32_t color){
```

Comment:
If you're calling this setIndicatorMode (which I think is a great name), I think the input parameter should be something that explicitly says eureka mode or autonomous mode or whatever, and then the function's internal logic decides the colour based on the input parameter.

You have a few options here:
- void setIndicatorMode(String indicatorMode) : you then check for the different cases and choose the colour internally
- void setIndicatorMode(int indicatorMode) : you could create macros or constants or an enum on top for the different colours, something like `int INDICATOR_RED = 1;` or `#define INDICATOR_RED 1` or `enum INDICATOR_COLORS{RED, GREEN, BLA}` not sure if my syntax is perfect. and then pass them into the function and the function recognizes them in a switch statement and uses that to choose the right colour
- void setIndicatorMode(uint32_t *indicatorMode) : you could create constant arrays with appropriate variable names (INDICATOR_RED_COLOR etc for example) near the top for the 3 activity indicator colours, then pass a pointer to the relevant array in this function and send it directly into setPixelColor()